plugins/frontend/backstage-plugin-argo-cd/src/components/useAppDetails.ts

```diff
@@ -140,7 +140,7 @@ export const useAppDetails = ({
           },
         );
         const result = await Promise.all(getRevisionHistroyPromises);
-        return result;
+        return result.filter(n => n);
```

Comment:
This is already a promise so there is need to `await` this. This is effectively `return await` which is redundant. 

```suggestion
        return Promise.all(getRevisionHistroyPromises)
          .then(result => result.filter(n => n))
        ;
```

Reply:
Thanks for your suggestions, couldn't commit from here, so pushed from my branch